### ISSUE TITLE
Cannot run from Ftbapp client

### ISSUE THREAD

**Issue by Angel-Tube** [LunaPixelStudios/Better-Minecraft#406]
### Describe the Bug

I installed better Minecraft 1.18.2 forge on Ftblauncher but it crashes before launch or stalls. I've already spoken with the ftb support and they directed me here.

### Reproduction Steps

1.install ftblauncher
2.install better Minecraft forge 1.18.2 modpack on ftblauncher
3.run better Minecraft forge 1.18.2
...

### Operating System

Pop!OS

### Minecraft Version

1.18

### Modloader

Forge

### Modpack Version

v11

### Optifine

No

**Comment by SharkieTV** [comment]
I need the latest.log also never used or heard of the ftblauncher tbh so

**Comment by Angel-Tube** [comment]
sorry, "ftbapp".  The feed the beast launcher which has curseforge modpacks. This is what the latest.log i have. The last time I tried launching the modpack before grabbing the log, it just stalled.
[latest.log](https://github.com/LunaPixelStudios/Better-Minecraft/files/8784243/latest.log)

**Comment by SharkieTV** [comment]
Can you try the Curseforge launcher?

**Comment by Angel-Tube** [comment]
Unfortunately, Curseforge still doesn't have minecraft linux support. The lutris workaround installers fail to install it. 
I have also been attempting alternative launchers available, such as ftbapp, gdlauncher, and atlauncher (not to be confused with tlauncher). ftbapp and atlauncher are able to install and launch other modpacks, but this modpack has been crashing consistently when attempting to launch through these. While GDlauncher is just currently having curseforge issues in general when I attempted install.
Alternatives of these are still unknown at the moment.
I wouldn't be in this mess if the MultiMC devs didn't drop the ball on picking up the new Curseforge API stuff and my aternos server didn't autoupdate. Which even my friend that has been able to launch the latest update has been crashing when trying to connect to it, but that is a different issue.

**Comment by Angel-Tube** [comment]
Alright. I want to profusely apologize for bothering you any. I managed to get the modpack launched through the atlauncher after some help from their support team. As far as I can tell it is a launcher client issue in relation to the modpack rather than just the modpack itself. I managed to get it to launch by checking off whatever this "feral mode" is in the atlauncher client settings. While it is still true that I can not get this modpack to launch through ftbapp, I have found an alternative in the meantime, but still willing to provide info where I can if wanted.
I am finally able to have a chance to try to connect to my server here shortly to see if my servermate is the only one not able to connect to the updated server. If any issues arise, I will open ticket accordingly at a later time regarding it. 
Thank you for your response and thank you for the modpack. I have never played minecraft this much before it.
